web/components/feed/feed-items.tsx

```diff
@@ -221,6 +226,31 @@ export function FeedComment(props: {
     money = formatMoney(Math.abs(matchedBet.amount))
   }
 
+  const [highlighted, setHighlighted] = useState(false)
+  const [showToast, setShowToast] = useState(false)
+  const router = useRouter()
+  useEffect(() => {
+    if (router.asPath.includes(`#${comment.id}`)) {
+      setHighlighted(true)
+    }
+  }, [router.asPath])
+
+  function copyLinkToComment(
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>
+  ) {
+    event.preventDefault()
+
+    let currentLocation = window.location.href.includes('/home?u=')
```

Comment:
Nice detail!

Reply:
😠 we need to get rid of the `/home?u=` thing